libs/ui/src/tokens/typography.ts

```diff
@@ -0,0 +1,172 @@
+type Typography = {
+    [key: string]: {
+            [key: string]: {
+                fontFamily: string;
+                fontSize: string;
+                fontWeight: number;
+                lineHeight: number;
+                letterSpacing?: string;
+            };
+    };
+};
+
+const typography: Typography =  {
+    large: {
+        'Headline 1': {
```

Comment:
I'm thinking we should slightly rename the keys to be easier to use, since they'll be used as tokens, but we can leave that for a future enhancement.